elasticdl/docker/build_all.sh

```diff
@@ -25,4 +25,4 @@ docker build -t $dev_image -f elasticdl/docker/Dockerfile.dev --build-arg BASE_I
 
 docker build -t $image -f elasticdl/docker/Dockerfile --build-arg BASE_IMAGE=$base_img .
```

Comment:
Can we update base_img -> base_image, keep the same naming convenstion

Reply:
Done